test/odr_test.py

```diff
@@ -1,48 +1,69 @@
+from __future__ import print_function, division
+
 import os
 import sys
 import re
+import glob
+
+try:
+    from pathlib import Path
+except ImportError:
+
+    # Drop when Python2 support is dropped
+    class Path(str):
```

Comment:
This part completely goes away when Python 2 is dropped, and allows it to be written in nicer Python 3 syntax. But okay.